### ISSUE TITLE
77 tokens limit ?

### ISSUE THREAD

**Issue by Cariboost** [rupeshs/fastsdcpu#261]
Hello,

I am using fastsdcpu on Android (btw I am using android 12 (on note 10+) even if compatibility says android 13 at least and it is working pretty well, even if quality is quite low but I didn't deep dive yet on why).

I am facing the limit of 77 tokens in the prompt : everything after 77 characters is ignored (terminal explicitly say that).

I check on Google and it seem to be quite a common issue, and most GUI like A1111 have solved it. 
I was wondering if a way existed with fastsdcpu ? Or is it something wanted to avoid CPU overload

**Comment by rupeshs** [comment]
yes current default token limitation is 77 but it is possible to extend it using compel library.

**Comment by mykeehu** [comment]
Here are some solutions. I'm no programmer, but maybe someone will make a PR after this:
https://github.com/huggingface/diffusers/issues/2136

**Comment by kotoji** [comment]
"compel" could be a good option, as introduced here:  
https://huggingface.co/docs/diffusers/using-diffusers/weighted_prompts#stable-diffusion-xl

**Comment by setoelkahfi** [comment]
I got this issue recently and changing my pipeline to this like the comment mentioned fixes the issue:

```python
pipe = DiffusionPipeline.from_pretrained(
    'hakurei/waifu-diffusion',
    custom_pipeline="lpw_stable_diffusion",
    torch_dtype=torch.float16
)
```
